src/start.js

```diff
@@ -2,13 +2,11 @@ import express from 'express'
 import 'express-async-errors'
 import logger from 'loglevel'
 import {getRoutes} from './routes'
-import {getRoutesTwo} from './routes'
 
 function startServer({port = process.env.PORT} = {}) {

```

Comment:
Consider doing something like this
```suggestion
function startServer({port = process.env} = {}) {

```